README.md

```diff
@@ -108,145 +40,59 @@ To proceed, click on the **Create a Bot** button and fill up all the required in
+Clone the git repository to your local machine by running the following command on your terminal.
 
 ```sh
-git clone --recursive https://github.com/haskalpa/docker-webexteams-bot-example.git
+git clone https://github.com/sarupanda/webexteams-python-example.git
 ```
 
-If you run the command `ls`, you will find all the directories and files inside your current directory. To go to directory you have just cloned simply run `cd docker-webexteams-bot-example`. Try running `ls` and check that you can see all the files you will need to get your Bot up and running.
-
-### 4. Configure with Access Token
-
-Remember the access token for the bot that you saved earlier? Here is where you will need it.
-
-Open the folder that you have just created, **docker-webexteams-bot-example**. Inside you will find a folder named **config**, open it. Then, create a copy of the **config_example.yaml** file and rename it to **config.yaml**.
-
-Open the new file with your favourite text editor. If you still do not have one, take a quick look at [Atom](https://atom.io/), [Visual Studio Code](https://code.visualstudio.com/) or (in case you are a very brave developer) [Vim](https://www.vim.org). This is what the file contents should look like:
+To go to directory you have just cloned simply run `cd webexteams-python-example`. Try running `ls` and check that you can see all the files you will need to get your Bot up and running.
 
-````sh
----
-hello_bot:
-teams_access_token: <my_bot_access_token>
-````
-
-Yep, you guessed it! Replace `<my_bot_access_token>` with the Access Token you save during the **Create a Bot** step.
-
-### 5. Build Docker container
-
-Back to the terminal window: run the following command to build your Docker container.
+After installing Python, open terminal and run the following commands to install dependencies.
 
 ```sh
-./build.sh
+pip3 install -r requirements.txt
 ```
 
-After a couple of seconds, you should get a `Successfully built` message in your terminal window.
-
 <a name="run"></a>
 ## Run Bot
 
-### 1. Run Docker container
-
-If you are a **Windows user**, you will need to start with **Step 1**. If not, just skip this step and go directly to **Step 2**.
-
-**Step 1**
-
-Open **Notepad++** and open the file **run_hello_bot.sh**. You don't have Notepad++? Just quickly install it [here](https://notepad-plus-plus.org/).
-
-Once you have opened the file, go to the **Edit** menu on the top bar and click **EOL Conversion**. Set this to **Unix (LF)**. Then, save the file and close Notepad++.
-
-Repeat the above with **hello_bot/run.sh**.
+#### 1. Run Ngrok
 
-**Step 2**
-
-Run the Docker container with the following command.
+Copy the ngrok executable file that you downloaded above to the `webexteams-python-example` folder. On a terminal window, go to this directory and run the following command to expose a web server on port 5000 of your local machine to the internet.
```

Comment:
Should we mention that the executable is inside a zip and should be unzipped?

Reply:
will add this

Reply:
please check the changes here!

Reply:
Looks good!